contrib/labstack/echo/echotrace_test.go

```diff
@@ -0,0 +1,130 @@
+package echo
+
+import (
+	"errors"
+	"net/http/httptest"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/labstack/echo"
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
```

Comment:
Left over?

Reply:
actually, not necessary.  i'll remove it.